static/src/javascripts/projects/common/utils/closest.js

```diff
@@ -0,0 +1,18 @@
+define(function () {
+    /**
+     * Adds a wrapper around Element.closest()
+     */
+    var matches = 'matches' in Element.prototype ? 'matches' : 'msMatchesSelector';
+    return 'closest' in Element.prototype ? closestNative : closestPolyfill;
+
+    function closestNative(element, selectors) {
+        return element.closest(selectors);
+    }
+
+    function closestPolyfill(element, selectors) {
+        while (element && !element[matches](selectors)) {
```

Comment:
This is indeed counter-intuitive, but it is [how the `closest` method works](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest). ECMAScript authors would be lambasted by taxonomists, that's for sure 😆